### PR TITLE
replay: Fix blank line display for "-f none" option.

### PR DESCRIPTION
**Pull request by ekyooo**
Remove unnecessary blank line display of replay with "-f none" option.
test116 is passsed

Signed-off-by: Eunseon Lee <esintospace@gmail.com>

### CODE REVIEW

File: cmds/replay.c [hunk 1]
```diff
@@ -1183,10 +1183,11 @@ int command_replay(int argc, char *argv[], struct opts *opts)
...
 		print_header(&output_fields, "#", "FUNCTION", 1, false);
-	if (opts->srcline)
-		pr_gray(" [SOURCE]");
-	pr_out("\n");
-
+	if (!list_empty(&output_fields)) {
+		if (opts->srcline)
+			pr_gray(" [SOURCE]");
+		pr_out("\n");
+	}
```
Comment (namhyung) [inline cmds/replay.c:1190]: Can you please keep the blank line before the while loop below?  It'd separate the header line from data.

Comment (ekyooo) [reply cmds/replay.c:1190]: sorry. it's updated.

**Comment by namhyung** [comment]
I'm sorry but I've found Honggyu already sent a patch for it in #1170.

**Comment by honggyukim** [comment]
I’m okay to apply this instead of mine as it’s included many other patches so will take some more time to be merged. I can manage my PR once this patch is merged before mine.

**Comment by ekyooo** [comment]
It's nice. thank you @honggyukim.
I close this.

**Pull request closed without merging**